tests/integration/test_vulnerability_detector/test_providers/test_providers_update_interval.py

```diff
@@ -58,13 +80,19 @@
 # Fixtures
 @pytest.fixture(scope='module', params=configurations, ids=ids)
 def get_configuration(request):
-    """Get configurations from the module."""
+    """test_logic:
```

Comment:
As this isn´t a test method it isn´t required to follow the DocGenerator format.
But to be aligned, it´s a good practice to migrate this under YAML format, but, I think "brief" will be the correct key here.